install/pre-install-payload/payload.sh

```diff
@@ -57,6 +57,7 @@ function build_payload() {
 
 	tag=$(git rev-parse HEAD)
 
+  manifest_args=()
```

Comment:
nit: mind the gap

Reply:
ah, didn't noticed this, @beraldoleal could you please address it, then it should be ready to be merged.